gremlin-dotnet/src/Gremlin.Net/Process/Traversal/GraphTraversal.cs

```diff
@@ -1115,6 +1124,15 @@ private GraphTraversal(ICollection<ITraversalStrategy> traversalStrategies, Byte
             Bytecode.AddStep("label");
             return Wrap<TStart, string>(this);
         }
+        
+        /// <summary>
+        ///     Adds the length step to this <see cref="GraphTraversal{SType, EType}" />.
+        /// </summary>
+        public GraphTraversal<TStart, int> Length ()
```

Comment:
Sorry, but I didn't notice this during my first review of this PR: Since `length()` returns `null` as `null`, this should return `int?` instead of `int`. This communicates that this step can return either an `int` or `null`. (This of course also applies to `__.Length()`.

Reply:
Thank you for all the reviews Florian! I will fix these as needed.